modules/nw-externalip-about.adoc

```diff
@@ -10,7 +10,7 @@ For non-cloud environments, {product-title} supports the assignment of external
 By setting this field, {product-title} assigns an additional virtual IP address to the service. The IP address can be outside the service network defined for the cluster.
 A service configured with an ExternalIP functions similarly to a service with `type=NodePort`, allowing you to direct traffic to a local node for load balancing.
 
-You must configure your networking infrastructure to ensure that the external IP address blocks that you define are routed to the cluster.
+You must configure your networking infrastructure to ensure that the external IP address blocks that you define are routed to the cluster. As a result, the IP address is not configured in the network interfaces from nodes. To handle the traffic, the administrator must configure the routing and access to the external IP by using a method such as static Address Resolution Protocol (ARP) entries.
```

Comment:
Is "the administrator" in this sentence going to be the audience of this doc and/or the same person who performs the action in the first sentence of this line? If so, I'd suggest changing that to say "you" to have a greater user focus:
```suggestion
You must configure your networking infrastructure to ensure that the external IP address blocks that you define are routed to the cluster. As a result, the IP address is not configured in the network interfaces from nodes. To handle the traffic, you must configure the routing and access to the external IP by using a method such as static Address Resolution Protocol (ARP) entries.
```

Up to you though